xml/containers-troubleshoot.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha-containers-troubleshoot"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Troubleshooting</title>
+ </info>
+ <sect1 xml:id="sec-container-diff">
+  <title>Analyze Container Images with container-diff</title>
+
+  <p>
+  In case a custom &deng; container image built on top of the &slea; base container image is not working as expected, the <package>container-diff</package> tool can help you analyze the image and collect information relevant for troubleshooting.
```

Comment:
```suggestion
  In case a custom &deng; container image built on top of the &slea; base container
  image is not working as expected, the <package>container-diff</package> tool can
  help you to analyze the image and collect information relevant for troubleshooting.
```